### DISCUSSION TITLE
new README file

### DISCUSSION THREAD

**Discussion by CrO2Cl2** [TriliumNext/Notes#59] [Ideas]
I think that remaking the README file should be pretty far up on our priorities. I think that just replacing every Trilium with Triliumnext is not enough, and we should remake it so that it still shows that we are tied to trilium, but still be different. 
I think that a discussion is needed to decide how the readme file should change, what we should add or remove

**Comment by alexpietsch** [comment]
Yes, I agree. There are also a lot of references in the code that we need to replace.

**Reply by CrO2Cl2** [reply]
for sure. another thing that we have to think of, is wether to replace the email for the security issues. Do we still leave that up to Zadam? but if the sec-issue is fault of something that we did, then he won't have a clue about it.

**Reply by meichthys** [reply]
I think it should be updated or removed in favor of a chat channel. Obviously we would report any issues to Trilium if it is relevant.

**Reply by CrO2Cl2** [reply]
wasn't the idea of using email for very bad sec issues that no other people can read it and start to abuse it?

**Reply by meichthys** [reply]
Yes, I think so. It should be possible via chat too with DMs or perhaps a channel that is only visible to a special security group.

**Reply by CrO2Cl2** [reply]
I do still think email would be a better medium, as not everyone has matrix or XMPP, but everyone has an email.

**Reply by meichthys** [reply]
That's true! It would just require a common email for devs to monitor, or one person designated to receive such emails.

**Comment by Byteingpython** [comment]
The README and the Organisation description should be also updated to communicate that the original zadam/trilium is entering maintenance mode, not Trilium Next. At least for me that wasn't obvious at first

**Comment by shutaozhenzhen** [comment]
By the way, if we want to change things in the Readme, should we remove the support Ukraine?

**Reply by CrO2Cl2** [reply]
I would leave it, but perhaps put it a bit further down

**Reply by Byteingpython** [reply]
I think we should at least remove the picture, as being greeted by an artillery shell isn't a very friendly greeting

**Comment by FliegendeWurst** [comment]
Related issue about the same topic: https://github.com/TriliumNext/Notes/issues/1816

I proposed some changes to the readme: https://github.com/TriliumNext/Notes/pull/1976
